classes/Cart.php

```diff
@@ -3918,17 +3866,17 @@ public function getSummaryDetails($id_lang = null, $refresh = false)
             'invoice_state' => State::getNameById($invoice->id_state),
             'formattedAddresses' => $formatted_addresses,
             'products' => array_values($products),
-            'gift_products' => $gift_products,
-            'discounts' => array_values($cart_rules),
+            'gift_products' => [],
```

Comment:
I don't think we need this field for the raw return, it should only be added in `getSummaryDetails`